README.md

```diff
@@ -1,17 +1,65 @@
+Some scenarios in which you may find go-libipfs helpful:
 
-## Contributing
+* You are building an application that interacts with the IPFS network
+* You are building an IPFS implementation
+* You want to reuse some components of IPFS such as its Kademlia DHT, Bitswap, data encoding, etc.
+* You want to experiment with IPFS
+
+Go-libipfs powers [Kubo](https://github.com/ipfs/kubo), which is the most popular IPFS implementation, so its code has been battle-tested on the IPFS network for years, and is well-understood by the community.
+
+## What kind of components does go-libipfs have?
+
+Go-libipfs includes high-quality components useful for interacting with IPFS protocols, public and private IPFS networks, and content-addressed data, such as:
+
+- Content routing (DHT, delegated content routing, providing)
+- Data transfer (gateways, Bitswap, incremental verification)
+- Naming and mutability (name resolution, IPNS)
+- Interacting with public and private IPFS networks
+- Working with content-addressed data
+
+Go-libipfs aims to provide a cohesive interface into these components. Note that not all of the underlying components necessarily reside in this respository.
+
+## Getting started
+TODO
+
+## Should I add my IPFS component to go-libipfs?
+We happily accept external contributions! However, go-libipfs maintains a high quality bar, so code accepted into go-libipfs must meet some minimum maintenance criteria:
+
+* Actively maintained
+  * Must be actively used by, or will be included in software that is actively used by, a significant number of users or production systems. Code that is not actively used cannot be properly maintained.
+  * Must have multiple engineers who are willing and able to maintain the relevant code in go-libipfs for a long period of time.
+  * If either of these changes, go-libipfs maintainers will consider removing the component from go-libipfs.
+* Adequately tested
+  * At least with unit tests
+  * Ideally also including integration tests with other components
+* Adequately documented
+  * Godocs at minimum
+  * Complex components should have their own doc.go or README.md describing the component, its use cases, tradeoffs, design rationale, etc.
+* If the maintainers are not go-libipfs maintainers, then the component must include a CODEOWNERS file with at least two code owners who can commit to reviewing PRs
+
+If you have some experimental component that you think would benefit the IPFS community, we suggest you build the component in your own repository until it's clear that there's community demand for it, and then open an issue in this repository to discuss including it in go-libipfs.
+
+## Help
+
+If you have questions, feel free to open an issue. You can also find the go-libipfs maintainers in [Slack](https://filecoin.io/slack/) at #go-libipfs-maintainers.
```

Comment:
Fine for now, but we can also add IPFS Discord and ipfs.io Matrix to make it easier for people to meet us where they are.